### ISSUE TITLE
The appimage application does not open in NixOS

### ISSUE THREAD

**Issue by Inarizxc** [DIDIRUS4/AstralRinth#47]
The terminal outputs:
```
** (ModrinthApp:86703): WARNING **: 18:09:39.342: atk-bridge: get_device_events_reply: unknown signature
```
It started the first time and then stopped.

**Comment by Inarizxc** [comment]
Here's my file for creating the derivation:
```
{appimageTools, fetchurl, ...}:
let
  pname = "ModrinthApp";
  version = "0.8.9";

  src = fetchurl {
    url = "https://github.com/DIDIRUS4/AstralRinth/releases/download/ARF-v${version}/AstralRinth.App_${version}_amd64.AppImage";
    sha256 = "sha256-HYb3kYfYjRTqHv4zVw5xWXELyVC2ATivf2ebvvKadx8=";
  };

  appimageContents = appimageTools.extractType2 {
    inherit pname version src;
  };
in
appimageTools.wrapType2 rec {
  inherit pname version src;

  extraInstallCommands = ''
    mkdir -p $out/share/applications $out/share/pixmaps
    # cp ${appimageContents}/AstralRinth\ App.desktop $out/share/applications/
    cp ${appimageContents}/AstralRinth\ App.png $out/share/pixmaps/

    touch $out/share/applications/AstralRinth\ App.desktop

    cat > $out/share/applications/AstralRinth\ App.desktop <<EOF
    [Desktop Entry]
    Categories=Game;
    Exec=nvidia-offload ModrinthApp
    Icon=ModrinthApp
    Name=AstralRinth App
    Terminal=false
    Type=Application
    MimeType=application/zip+mrpack;x-scheme-handler/modrinth

    # for n in {16,32,48,64,128}; do
    #   size=$n"x"$n
    #   mkdir -p $out/share/icons/hicolor/$size/apps
    #   file="default"$n".png"
    #   cp -r ${appimageContents}/usr/share/* $out/share/
    # done

    # substituteInPlace $out/share/applications/NeoHtop.desktop \
    #   --replace-fail "Exec=neohtop %u" "Exec=$out/bin/${pname} %u"
  '';
}
```

**Comment by Inarizxc** [comment]
The application starts if you delete the app directory

**Comment by DIDIRUS4** [comment]
Unfortunately, I am not adding any new functionality, but only support offline access and removal of ads to get a better gaming experience. Perhaps your problem has already been solved in v0.9.2 from Modrinth or in v0.9.201-dev from AstralRinth
